Description: Manpage adjustments
 This patch adjusts the manpage by specifying further clock designs included in
 the Debian distribution.
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=345430

--- swisswatch-0.6.orig/swisswatch.man
+++ swisswatch-0.6/swisswatch.man
@@ -205,11 +205,17 @@ If set, the clock will remain circular e
 to a non-circular ellipse.  This is the previous behavior.  The
 possible values are the same as for \fB.swiss/.Swiss\fP.
 .SH EXAMPLE RESOURCES
-The distributed application defaults files define three different
+The distributed application defaults files define various different
 looks for swisswatch.  Try them out by calling it as
 .sp 1
 swisswatch
 .sp 1
+swisswatch -name swisswatch
+.sp 1
+swisswatch -name fancy
+.sp 1
+swisswatch -name botta
+.sp 1
 swisswatch -name swissclock
 .sp 1
 swisswatch -name oclock

